doc/release-notes/6.2-release-notes.md

```diff
@@ -0,0 +1,317 @@
+
+### Files Endpoint Extended
+
+The API endpoint `api/files/{id}` has been extended to support the following optional query parameters:
+
+- **includeDeaccessioned**: Indicates whether or not to consider deaccessioned dataset versions in the latest file search. (Default: `false`).
+- **returnDatasetVersion**: Indicates whether or not to include the dataset version of the file in the response. (Default: `false`).
+
+A new endpoint `api/files/{id}/versions/{datasetVersionId}` has been created. This endpoint returns the file metadata present in the requested dataset version. To specify the dataset version, you can use `:latest-published`, `:latest`, `:draft` or `1.0` or any other available version identifier.
+
+The endpoint supports the *includeDeaccessioned* and *returnDatasetVersion* optional query parameters, as does the `api/files/{id}` endpoint.
+
+`api/files/{id}/draft` endpoint is no longer available in favor of the new endpoint `api/files/{id}/versions/{datasetVersionId}`, which can use the version identifier ``:draft`` (`api/files/{id}/versions/:draft`) to obtain the same result.
+
+### Datasets, Dataverse Collections, and Datafiles Endpoints Extended
+
+The API endpoints for getting datasets, Dataverse collections, and datafiles have been extended to support the following optional 'returnOwners' query parameter.
+
+Including the parameter and setting it to true will add a hierarchy showing which dataset and dataverse collection(s) the object is part of to the json object returned. 
+
+### Endpoint Fixed: Datasets Metadata
+
+The API endpoint `api/datasets/{id}/metadata` has been changed to default to the latest version of the dataset that the user has access.
+
+
+## 📖 Guides
+
+### New QA Guide
+
+A new QA Guide is intended mostly for the core development team but may be of interest to contributors.
+
+### Container Guide, Documentation for Faster Redeploy
+
+In the Container Guide, documentation for developers on how to quickly redeploy code has been added for Netbeans and improved for IntelliJ.
+
+Also in the context of containers, a new option to skip deployment has been added and the war file is now consistently named "dataverse.war" rather than having a version in the filename, such as "dataverse-6.1.war". This predictability makes tooling easier.
+
+Finally, an option to create tabs in the guides using [Sphinx Tabs](https://sphinx-tabs.readthedocs.io) has been added. (You can see the tabs in action in the "dev usage" page of the Container Guide.) To continue building the guides, you will need to install this new dependency by re-running  `pip install -r requirements.txt`.
+
+### Evaluation Version Tutorial on the Containers Guide
+
+The Container Guide now containers a tutorial for running Dataverse in containers for demo or evaluation purposes: https://guides.dataverse.org/en/6.2/container
```

Comment:
I'm just leaving this not on the last line of the release notes but we need the rest of it. 😄 

We especially need the upgrade instructions. That's the hardest part.